modules/xz/metadata.json

```diff
@@ -0,0 +1,17 @@
+{
+    "homepage": "https://tukaani.org/",
+    "maintainers": [
+        {
```

Comment:
@Attempt3035 Can you please confirm you are fine with being a maintainer for this module?

Reply:
Yep, happy to be maintainer!

Reply:
Thanks!